sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/ExpressionEvalHelper.scala

```diff
@@ -109,7 +109,7 @@ trait ExpressionEvalHelper {
     }
 
     val actual = plan(inputRow)
-    val expectedRow = new GenericRow(Array[Any](CatalystTypeConverters.convertToCatalyst(expected)))
+    val expectedRow = InternalRow.fromSeq(Array(CatalystTypeConverters.convertToCatalyst(expected)))
```

Comment:
@davies , we need the conversion here, as we use `String` instead of `UTF8String` at many tests...